tests/integration/test_dbapi_integration.py

```diff
@@ -32,6 +32,15 @@ def trino_connection(run_trino):
     )
 
 
+@pytest.fixture
+def trino_connection_jmx(run_trino):
```

Comment:
Not sure if you need to add this new connection. See Trino docs: https://trino.io/docs/current/sql/set-role.html

Reply:
you're right. Not needed